src/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -575,5 +575,10 @@ public override Task WriteAsync(ReadOnlyMemory<byte> source, CancellationToken c
         {
             return _sslState.SecureStream.WriteAsync(source, cancellationToken);
         }
+
+        public override Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken)
+        {
```

Comment:
What about the Memory-based overload? Are you doing that separately?

Reply:
Was going to but it makes sense to do it right now. So its done :)